config/snowflake.spc

```diff
@@ -16,6 +16,10 @@ connection "snowflake" {
   # If not specified, the default role for the user will be used
   # role = "ACCOUNTADMIN"
 
+  # Specifies the sSnowflake warehouse to use for executing snowflake queries (Required for schemata, session and login_history tables)
+  # If not specified, the default warehouse for the user will be used
+  # warehouse = "COMPUTE_WH"
```

Comment:
How many warehouses can you have in a Snowflake account, one or many? If you can have multiple, is it restricting if we restrict one warehouse per connection for the tables that use them? And would we eventually have a `snowflake_warehouse` table as well?